opendevin/runtime/docker/ssh_box.py

```diff
@@ -199,7 +199,7 @@ class DockerSSHBox(Sandbox):
     def __init__(
         self,
         container_image: str | None = None,
-        timeout: int = 120,
+        timeout: int = 180,
```

Comment:
Isn't this the timeout after which a background command is killed? The default of 120 seems quite high since the app looks hung. 😔 But maybe I'm not seeing this right, I'm looking on the web interface for the moment

Reply:
Exactly. The default timeout shows up in issues like https://github.com/OpenDevin/OpenDevin/issues/1280 and not only for interactive programs, any command that the agent runs, which doesn't return fast. The app looks stuck from the user perspective, for the duration.

That said, I thought this init is called normally with a timeout, set by config.sandbox_timeout, so the default here won't make a difference. So I'm curious why you set it higher.